src/parse/text/simple_extensions/mod.rs

```diff
@@ -9,6 +9,9 @@ use crate::{
     text,
 };
 
+#[allow(missing_docs)]
```

Comment:
Nit: can you add a simple doc string to the module?

Reply:
Added in [commit](https://github.com/substrait-io/substrait-rs/pull/185/commits/42738efd2bab1e5f961594ddfe30d02d5ff0c52d).